src/web/client/utilities/graphClientUtils.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import path from "path";
+import WebExtensionContext from "../WebExtensionContext";
+import { getCommonHeaders } from "../common/authenticationProvider";
+import * as Constants from "../common/constants";
+import { telemetryEventNames } from "../telemetry/constants";
+
+export async function requestGraphClient(graphToken: string, service: string, userId: string) {
```

Comment:
this function can be names better - you are making a network call here and fetching a repsonse. The name suggests this is a call to get client - that will be used futher for subsequent flow.